### PR TITLE
[fix](udf) fix java-udf coredump as get env return nullptr

### PR DESCRIPTION
**Pull request by zhangstar333**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by zhangstar333** [comment]
run buildall

File: be/src/runtime/fold_constant_executor.cpp [hunk 1]
```diff
@@ -122,7 +122,7 @@ Status FoldConstantExecutor::fold_constant_vexpr(const TFoldConstantParams& para
         }
         expr_result_map->insert({m.first, pexpr_result_map});
     }
-
+    LOG(WARNING) << "finish fold_query_id: " << query_id_string();
```
Comment (yiguolei) [inline be/src/runtime/fold_constant_executor.cpp:125]: why  it is a warning log???

**Comment by zhangstar333** [comment]
run buildall

**Review by Gabriel39 — DISMISSED** [review]

File: be/src/runtime/fold_constant_executor.cpp [hunk 2]
```diff
@@ -122,7 +122,7 @@ Status FoldConstantExecutor::fold_constant_vexpr(const TFoldConstantParams& para
         }
         expr_result_map->insert({m.first, pexpr_result_map});
     }
-
+    LOG(INFO) << "finish fold_query_id: " << query_id_string();
```
Comment (HappenLee) [inline be/src/runtime/fold_constant_executor.cpp:125]: the log really need ？ or just for debuging？

**Comment by zhangstar333** [comment]
run buildall

**Review by HappenLee — APPROVED** [review]
LGTM

**Review by Gabriel39 — APPROVED** [review]

**Pull request merged**